src/modes.cpp

```diff
@@ -207,8 +207,8 @@ void aste::runMapperMode(const aste::ExecutionContext &context, const OptionMap
 
   auto asteInterface = asteConfiguration.asteInterfaces.front();
   VLOG(1) << "Loading mesh from " << asteInterface.meshes.front().filename();
-  ;
-  asteInterface.meshes.front().loadMesh(asteInterface.mesh, dim);
+  const bool requireConnectivity = preciceInterface.isMeshConnectivityRequired(asteInterface.meshID);
+  asteInterface.meshes.front().loadMesh(asteInterface.mesh, dim, requireConnectivity);
   asteInterface.meshes.front().loadData(asteInterface.mesh);
   VLOG(1) << "The mesh contains: " << asteInterface.mesh.summary();
```

Comment:
Let's rephrase the VLOG to something more appropriate. Maybe `Configured the following mesh information for mesh <Name>: `

Reply:
Changed with 

`The loaded mesh <Name> contains: <summary>`